pkg/template/processor.go

```diff
@@ -1,15 +1,17 @@
 package template
 
 import (
+	"fmt"
 	"math/rand"
 	"time"
 
-	"github.com/codeready-toolchain/toolchain-common/pkg/client"
+	commonclient "github.com/codeready-toolchain/toolchain-common/pkg/client"
```

Comment:
same remark here about aliases, for the sake of consistency with other files ;)

Reply:
yeah, this should have been `applyclient`

Reply:
hmmm... not sure what should be the right alias for that - in other cases we use:
* `applycl`
* `commonclient`
* `commoncl`
* `applyclientlib`

:man_shrugging:

Reply:
let's keep it as it is and let's add it to the effort of aligning the import aliases.

Reply:
ok, sounds good. This is a cosmetic remark and not a blocker for this PR for sure, but if we could all agree on a policy for aliases, that would be great IMO